src/components/form/input.js

```diff
@@ -173,15 +179,18 @@ const Input = ({
     label_color,
     disabled,
     id,
+    isAffiliate,
+    isDate,
```

Comment:
All variables should be in snake_case, including booleans.

Reply:
Refactored all occurrences